homeassistant/util/location.py

```diff
@@ -138,11 +138,7 @@ def vincenty(
             / 4
             * (
                 cosSigma * (-1 + 2 * cos2SigmaM**2)
-                - B
-                / 6
-                * cos2SigmaM
-                * (-3 + 4 * sinSigma**2)
-                * (-3 + 4 * cos2SigmaM**2)
+                - B / 6 * cos2SigmaM * (-3 + 4 * sinSigma**2) * (-3 + 4 * cos2SigmaM**2)
```

Comment:
This harms readability, lets disable formatting around this part.

Reply:
That function doesn't adhere much to the surrounding code style either, but sure thing, adding some strategic `# fmt: ...`s :)